docker/Dockerfile.customopbuilder.clean

```diff
@@ -56,4 +56,11 @@ RUN rm -f /usr/bin/python && \
     pip --version
 
 COPY --from=cuda /usr/local/cuda /usr/local/cuda
-ENV LD_LIBRARY_PATH=/usr/local/cuda/lib64:/usr/local/cuda/lib64/stubs/:${LD_LIBRARY_PATH}
\ No newline at end of file
+ENV LD_LIBRARY_PATH=/usr/local/cuda/lib64:/usr/local/cuda/lib64/stubs/:${LD_LIBRARY_PATH}
+
+RUN export USE_CUDA_VERSION=$(cat /usr/local/cuda/version.txt | head -1 | sed 's/.*Version \([0-9]\+\)\.\([0-9]\+\).*/\1/') &&\
+    if [ "${USE_CUDA_VERSION}" = "9" ]; then export SUPPORTED_TF_VERSIONS="1.7.0 1.11.0 1.12.0"; fi && \
```

Comment:
I think, we should have single source of truth for supported TensorFlow versions. Now it is listed here and in `qa/setup_packages.py`.

Reply:
They doesn't have to match. In test we may have less (just a few old, and the most recent), here we need all that differs in ABI.

Reply:
Currently they are just copy-pasted.

Reply:
But if they are really independent, that's ok.

Reply:
Now they are:
- https://github.com/NVIDIA/DALI/blob/master/qa/setup_packages.py
- this docker file
- https://github.com/NVIDIA/DALI/blob/master/dali_tf_plugin/build_in_custom_op_docker.sh
Maybe one common file inside dali_tf_plugin that will be read/included in all other places could work.
What do you think?

Reply:
Done